pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -132,8 +132,10 @@ func defaultStrategyChooser(instance *v1alpha1.Jaeger) strategy.S {
 }
 
 func (r *ReconcileJaeger) apply(jaeger v1alpha1.Jaeger, str strategy.S) error {
-	if err := r.applyElasticsearches(jaeger, str.Elasticsearches()); err != nil {
-		return err
+	if strings.EqualFold(viper.GetString("es-provision"), v1alpha1.FlagProvisionElasticsearchTrue) {
```

Comment:
I would rather always check if the es operator is running and if it's not I would log an error here. 

* It's a simpler change
* it explicitly tells user something is wrong, whereas with the current approach logs a message at init time

Reply:
> check if the es operator is running

That's not really an easy thing to do. We can't know for sure *where* the operator is running. We should really just be checking if the cluster understands what a "Elasticsearch" CR is.

Reply:
I mean cluster understand's ES CR...